frontend/public/components/RBAC/bindings.jsx

```diff
@@ -419,7 +419,7 @@ class BaseEditRoleBindingWithTranslation extends React.Component {
       'subjects',
     ]);
     existingData.kind = props.kind;
-    const { subjectKind, subjectName } = this.props.fixed.subjectRef;
+    const { subjectKind, subjectName } = props.fixed.subjectRef || {};
```

Comment:
Should we expect `fixed` to always have a value?

Reply:
@spadgett Yes

Reply:
@cyril-ui-developer So I just tested this, and I get an error even when I edit a role binding that has a subject. I don't think the problem is the null guard... It seems that `subjectRef` is not being set when it should have a value.

Reply:
@spadgett The `subjectRef` was introduced for User and Group context. There is a  `subjects` object that is set for Role Binding and project pages. I have removed the `subjectRef` and bound the values with the `subjects` object. PTAL